admin/client/App/screens/List/index.js

```diff
@@ -339,7 +339,7 @@ const ListView = React.createClass({
 			confirmationDialog: {
 				isOpen: true,
 				label: 'Delete',
-				body: `Are you sure you want to delete <strong>${item.name}</strong>?<br /><br />This cannot be undone.`,
+				body: <p>Are you sure you want to delete <strong>${item.name}</strong>?<br /><br />This cannot be undone.</p>,
```

Comment:
That's pretty close to being multiline JSX, should we break it out and wrap it in `()` for readability?